dev/custom-pages/research.scss

```diff
@@ -380,6 +373,12 @@ $color-linkedin: #0077b5;
       .entry-title {
         margin-top: 0;
       }
+
+      @media (max-width: $screen-sm-max) {
+        img {
```

Comment:
Avoid nested selectors within media queries when using sass--something like
```
img {
  @media (max-width: $screen-sm-max) {
    ...declarations here...
  }
}
```
is preferred (per-selector media queries).